.gitignore

```diff
@@ -2,4 +2,3 @@ target
 .idea
 halodb.iml
 tmp/
-
```

Comment:
revert unnecessary change

Reply:
It maybe due to some IDE auto-edit. I think it's OK to keep this minor change here as we don't need space line in gitignore.